source/macos/AdaptiveCards/AdaptiveCards/Renderers/BaseCardElementRenderer.swift

```diff
@@ -11,7 +11,7 @@ class BaseCardElementRenderer {
         
         if !isfirstElement {
             // For seperator and spacing
-            updatedView.addSeparator(element.getSeparator(), withSpacing: element.getSpacing())
+            _ = SpacingView.renderSpacer(elem: element, forSuperView: updatedView, withHostConfig: hostConfig)
```

Comment:
Just to confirm
SpacingView internally adds separator?

Reply:
Yes